src/api/client.go

```diff
@@ -903,6 +903,20 @@ func (c *Client) PendingTransactionsVerbose() ([]readable.UnconfirmedTransaction
 	return v, nil
 }
 
+// GetTransactions makes a request to POST /api/v1/transactions
+func (c *Client) GetTransactions(addrs []string) ([]readable.TransactionWithStatus, error) {
```

Comment:
The `Transactions` method already does this

Reply:
Will take a look at it now.

Reply:
Changes have been pushed. See below.

Reply:
Wondering if using `TransactionsVerbose` would be more useful as it provides both `input` and `output` - whereas `Transactions` appears to just provide `output`

Reply:
The Verbose is probably better to use for this